_pages/home/updating-to-version-2211.35/modernize-apps-migrated-from-6.8-to-2211.19.md

```diff
@@ -1,466 +1,473 @@
 
-4. In the call `express.static( `... please use the constant `browserDistFolder` instead of `distFolder` as an argument.
+   ```ts
+   // Webpack will replace 'require' with - '__webpack_require__'
+   // '__non_webpack_require__' is a proxy to Node 'require'
+   // The below code is to ensure that the server is run - only when not requiring the bundle.
+   declare const __non_webpack_require__: NodeRequire;
+   const mainModule = __non_webpack_require__.main;
+   const moduleFilename = (mainModule && mainModule.- filename) || '';
+   if (moduleFilename === __filename || moduleFilename.- includes('iisnode')) {
+     run();
+   }
+   ```
 
-```diff
-   server.get(
-     '*.*',
--    express.static(distFolder, {
-+    express.static(browserDistFolder, {
-```
+1. In place of the block of code that was removed in the previous step, add the following line:
 
-5. Remove the block of code in the bottom of the file related handling Webpack's `require`, but leave only the call of the function `run()`
+   ```ts
+   run();
+   ```
 
-```diff
-- // Webpack will replace 'require' with - '__webpack_require__'
-- // '__non_webpack_require__' is a proxy to Node 'require'
-- // The below code is to ensure that the server is run - only when not requiring the bundle.
-- declare const __non_webpack_require__: NodeRequire;
-- const mainModule = __non_webpack_require__.main;
-- const moduleFilename = (mainModule && mainModule.- filename) || '';
-- if (moduleFilename === __filename || moduleFilename.- includes('iisnode')) {
--   run();
-- }
+1. At the very end of `server.ts`, remove the following line:
 
-+ run();
-```
+   ```ts
+   export * from './src/main.server';
+   ```
 
-6. In the very bottom of the file, remove the re-export of the path `./src/main.server`
+   You have now completed the migration of your app to use the Angular `application` builder. You can now continue with the following procedure.
 
-```diff
-- export * from './src/main.server';
-```
+## Using Non-Deprecated Angular APIs
 
-This is the end of the migration guide for the Angular `application` builder.
+After completing the migration of your app to use the new Angular `application` builder, it is also necessary to update the `app.module.ts` file to use new, non-deprecated Angular APIs.
 
-## In `app.module.ts`, use new, non-deprecated Angular APIs
+1. In `src/app/app.module.ts`, remove `HttpClientModule` from the `imports` array.
 
-Unrelated to the migration to the new Angular `application` builder, please also update the `app.module.ts` file to use new, non-deprecated Angular APIs:
+1. Add `provideHttpClient(withFetch(), withInterceptorsFromDi()),` to the `providers` array.
 
-### `src/app/app.module.ts`
+   The following is an example:
 
-1. Remove the `HttpClientModule` from the `imports` array. 
+   ```ts
+     providers: [
+       provideHttpClient(withFetch(), withInterceptorsFromDi()),
+     ],
```

Comment:
I'm sorry, it's a flaw in my doc input:
It's not the only providers there. So let's remove `]` or add `/*...*/` to not suggest that customers should remove all other items from array
```suggestion
```

Reply:
I added `/*...*/` 🙂